src/sidebar/util/build-thread.js

```diff
@@ -174,28 +174,11 @@ function mapThread(thread, mapFn) {
  * Return a sorted copy of an array of threads.
  *
  * @param {Thread[]} threads - The list of threads to sort
- * @param {(a: Annotation, b: Annotation) => boolean} compareFn
+ * @param {(a: Thread, b: Thread) => number} compareFn
  * @return {Thread[]} Sorted list of threads
  */
 function sort(threads, compareFn) {
-  return threads.slice().sort((a, b) => {
-    // Threads with no annotation always sort to the top
-    if (!a.annotation || !b.annotation) {
-      if (!a.annotation && !b.annotation) {
-        return 0;
-      } else {
-        return !a.annotation ? -1 : 1;
-      }
-    }
-
-    if (compareFn(a.annotation, b.annotation)) {
-      return -1;
-    } else if (compareFn(b.annotation, a.annotation)) {
-      return 1;
-    } else {
-      return 0;
-    }
-  });
+  return threads.slice().sort((a, b) => compareFn(a, b));
```

Comment:
Before, the sorting compare functions had no say in how headless annotations were sorted...

Reply:
Thank you, good point. Oversight from inherited code...